cmd/buildkitd/main.go

```diff
@@ -207,6 +212,18 @@ func main() {
 
 		controller.Register(server)
 
+		ents := c.GlobalStringSlice("allow-insecure-entitlement")
+		for _, e := range ents {
+			switch e {
+			case "security.unconfined":
+				llbsolver.AllowSecurityUnconfinedUnstable = true
```

Comment:
Don't use these variables anymore. Instead, add them to a config array and pass to the solver constructor.

Reply:
@kunalkushwaha